changelog/@unreleased/pr-4121.v2.yml

```diff
@@ -0,0 +1,7 @@
+type: fix
+fix:
+  description: |2
```

Comment:
you have a random `2` here, or is this intentional?

Reply:
Hmm, I didn't generate this file by hand! Maybe changelog bot is being weird